htpasswd.go

```diff
@@ -130,7 +162,14 @@ func (h *HtpasswdFile) Reload() error {
 
 	// Replace the Users map
 	h.mutex.Lock()
-	h.Users = users
+	var zeros [sha256.Size]byte
+	// try to wipe the old cache entries
```

Comment:
I don't think this is necessary since we only store hashed passwords in memory anyway.